app/views/additional_details/submissions/form_partials/_phone_form_partial.html.haml

```diff
@@ -1 +1 @@
-= qr.telephone_field :content, class: 'form-control'
+= qr.number_field :content, class: 'form-control'
```

Comment:
Would this still validate the format XXXXXXXXXX? Since it's a number field, wouldn't it allow any integer value?